### PR TITLE
rhbz#1182881 allow spaces in sub-man repos

### PR DESCRIPTION
**Pull request by mburns72h**

### CODE REVIEW

**Comment by bcrochet** [comment]
Visual ACK.

File: hooks/lib/provisioning_seeder.rb [hunk 1]
```diff
@@ -1061,7 +1061,7 @@ def redhat_register_snippet
     <% end %>
     # workaround for RHEL 6.4 bug https://bugzilla.redhat.com/show_bug.cgi?id=1008016
     subscription-manager repos --list > /dev/null
-    <%= "subscription-manager repos #{@host.params['subscription_manager_repos'].split(',').map { |r| '--enable=' + r.strip }.join(' ')}" if @host.params['subscription_manager_repos'] %>
+    <%= "subscription-manager repos #{@host.params['subscription_manager_repos'].split(/[\s,]/).map { |r| '--enable=' + r.strip }.join(' ')}" if @host.params['subscription_manager_repos'] %>
```
Comment (jistr) [inline hooks/lib/provisioning_seeder.rb]: The previous code was designed to handle cases like `repo-one, repo-two`, which this code wouldn't. Can we add a step to ignore empty strings from repositories? Also, we will not need the `.strip` call inside the map function now. Here's an example of what i'm thinking, the last line (006) shows what i think should be the new way of splitting the repo string:

```
2.0.0-p598 :001 > repos = 'repo-one, repo-two'
 => "repo-one, repo-two" 
2.0.0-p598 :002 > repos.split(/[\s,]/).map { |r| '--enable=' + r.strip }.join(' ')
 => "--enable=repo-one --enable= --enable=repo-two" 
2.0.0-p598 :003 > repos.split(/[\s,]/).reject { |r| r.empty? }.map { |r| '--enable=' + r.strip }.join(' ')
 => "--enable=repo-one --enable=repo-two" 
2.0.0-p598 :004 > repos = 'repo-one, repo-two    repo-three   ,    repo-four'
 => "repo-one, repo-two    repo-three   ,    repo-four" 
2.0.0-p598 :005 > repos.split(/[\s,]/).reject { |r| r.empty? }.map { |r| '--enable=' + r.strip }.join(' ')
 => "--enable=repo-one --enable=repo-two --enable=repo-three --enable=repo-four" 
2.0.0-p598 :006 > repos.split(/[\s,]/).reject { |r| r.empty? }.map { |r| '--enable=' + r }.join(' ')
 => "--enable=repo-one --enable=repo-two --enable=repo-three --enable=repo-four" 
```

**Comment by mburns72h** [comment]
thanks, updated

**Comment by jistr** [comment]
Thanks :)

**Pull request merged**